library-core/src/main/java/com/mikepenz/fastadapter/adapters/IItemList.java

```diff
@@ -0,0 +1,46 @@
+package com.mikepenz.fastadapter.adapters;
+
+import android.support.v7.util.ListUpdateCallback;
+import com.mikepenz.fastadapter.IItem;
+
+import java.util.Collection;
+import java.util.List;
+
+public abstract class IItemList<Item extends IItem> {

```

Comment:
Interface would be better.

Reply:
I used an abstract class because the `listCallback` is an implementation detail, and doesn't belong in the interface. 

A user creating a custom `IItemList` shouldn't be allowed to specify the `ListCallback`; that's something that `ModelAdapter` is in charge of.

Reply:
The idea is that there is an implementation that is using the ListUpdateCallback, but that doesnt mean every implementation has to use it.

Reply:
I don't see it as something that's worth exposing. It's just a replacement for calling the notify methods manually in each `ModelAdapter` call.

If it was exposed, there would need to be good docs explaining that no notify methods are gonna be called if you don't include them in your implementation.